amundsen_application/api/utils/notification_utils.py

```diff
@@ -80,34 +84,42 @@ def get_mail_client():  # type: ignore
     return mail_client
 
 
-def get_notification_content(*, notification_type: str, options: Dict) -> Dict:
+def get_notification_content(*, notification_type: str, options: Dict, recipients: List, sender: str) -> Dict:
     """
     Returns a subject and a rendered html email template based off
     the input notification_type and data provided
     :param notification_type: type of notification
     :param options: data necessary to render email template content
     :return: subject and html Dict
     """
+    user_id = recipients[0]
+    url = '{0}{1}/{2}'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id)
+
+    recipient_name = request_metadata(url=url).json()['first_name']
```

Comment:
I'm not a fan of having these extra requests here just to have personalized emails, it changes our call patterns in a way that we did not document.

I'd prefer to not shoot for personalized emails in this initial implementation anymore (or perhaps revisit next week if we have). The reason is that now we will want to consider calling all of these requests form the react application to begin with and passing in all parameters that are needed, or holding off on personalized emails until we have the support to do it the right way. (e.g. the owner information should ideally have first name and last name in the future not just email)